plugins/paper/src/main/java/com/playmonumenta/plugins/custominventories/ClassSelectionCustomInventory.java

```diff
@@ -608,26 +670,42 @@ public ItemStack createLevelItem(PlayerClass theClass, AbilityInfo<?> ability, i
 	public ItemStack createEnhanceItem(PlayerClass theClass, AbilityInfo<?> ability, Player player) {
 		if (ability.getDescriptions().size() == 3) {
 			Material newMat;
+			ItemStack newItem;
 			String scoreboard = ability.getScoreboard();
+			boolean selectedEn = false;
 			switch (scoreboard == null ? 0 : ScoreboardUtils.getScoreboardValue(player, scoreboard)) {
 				case 0 -> {
 					newMat = Material.BARRIER;
-					return GUIUtils.createBasicItem(newMat, 1,
+					ItemStack disabledEn = GUIUtils.createBasicItem(newMat, 1,
 						"Enhancement", theClass.mClassColor, true,
 						Component.text("Cannot Select; Choose levels in the ability first. Description: ").append(ability.getDescription(3)),
 						30, true);
+					GUIUtils.setGuiNbtTag(disabledEn, "texture", "skill_select_en_disabled");
+					return disabledEn;
+				}
+				case 1, 2 -> {
+					newMat = Material.ORANGE_STAINED_GLASS_PANE;
+					selectedEn = false;
+				}
+				case 3, 4 -> {
+					newMat = Material.YELLOW_STAINED_GLASS_PANE;
+					selectedEn = true;
 				}
-				case 1, 2 -> newMat = Material.ORANGE_STAINED_GLASS_PANE;
-				case 3, 4 -> newMat = Material.YELLOW_STAINED_GLASS_PANE;
 				default -> {
 					newMat = Material.BARRIER;
 					return GUIUtils.createBasicItem(newMat, "Unknown Level",
 						theClass.mClassColor, true, "Unknown level for ability.", NamedTextColor.WHITE);
 				}
 			}
-			return GUIUtils.createBasicItem(newMat, 1,
+			newItem = GUIUtils.createBasicItem(newMat, 1,
 				"Enhancement", theClass.mClassColor, true,
 				ability.getDescription(3), 30, true);
+			if (selectedEn) {
+				GUIUtils.setGuiNbtTag(newItem, "texture", "skill_select_en_lit");
+			} else {
+				GUIUtils.setGuiNbtTag(newItem, "texture", "skill_select_en_unlit");
+			}
+			return newItem;
```

Comment:
Can check the material type here instead, re-simplifying the case statement above